include/psp2kern/kernel/threadmgr/debugger.h

```diff
@@ -58,10 +58,10 @@ typedef struct SceThreadCpuRegisters
 /* Typedef for compatibility */
 typedef SceThreadCpuRegisters ThreadCpuRegisters;
 
-typedef struct SceKernelFaultingProcessInfo {
-	SceUID pid;
-	SceUID faultingThreadId; //!< Kernel UID of the faulting thread
-} SceKernelFaultingProcessInfo;
+typedef struct SceKernelThreadContextInfo {
+	ScePID processId;  //<! Process ID the thread scheduled on this CPU belongs to
+	SceUID threadId;   //<! Thread ID of the thread scheduled on this CPU
```

Comment:
```suggestion
	ScePID process_id;  //<! Process ID the thread scheduled on this CPU belongs to
	SceUID thread_id;   //<! Thread ID of the thread scheduled on this CPU
```
can you change it to `snake_case`?